packages/fxa-auth-server/lib/oauth/token.js

```diff
@@ -65,7 +65,11 @@ exports.verify = async function verify(accessToken) {
     // by slowly purging older tokens from the db.
     if (
       +token.expiresAt >=
-      config.get('oauthServer.expiration.accessTokenExpiryEpoch')
+        config.get('oauthServer.expiration.accessTokenExpiryEpoch') ||
```

Comment:
sort of a nit: as a casual reader the `||` makes it harder for me to follow. It'll probably be easier to understand, and to remove later, if we have a nested if.

```js
if (expired) {
  if (!pocket) {
   throw Error
```